sys/linux/sys.txt.const

```diff
@@ -608,6 +608,7 @@ __NR_fcntl = 55, amd64:72, arm64:riscv64:25, mips64le:5070
 __NR_fdatasync = 148, amd64:75, arm64:riscv64:83, mips64le:5073
 __NR_finit_module = 273, 386:350, amd64:313, arm:379, mips64le:5307, ppc64le:353, s390x:344
 __NR_flock = 143, amd64:73, arm64:riscv64:32, mips64le:5071
+__NR_fork = amd64:57
```

Comment:
Please add other arches as well.
`make extract` should update all arches properly.
If you don't have cross-compilers installed, consider using syz-env container:
https://github.com/google/syzkaller/blob/master/docs/contributing.md#using-syz-env

Reply:
OK, I will.